SpatialGDK/Source/SpatialGDK/Private/Utils/InterestFactory.cpp

```diff
@@ -382,4 +373,24 @@ QueryConstraint InterestFactory::CreateLevelConstraints() const
 	return LevelConstraint;
 }
 
+TArray<uint32> InterestFactory::CreateClientResultType() const
+{
+	TArray<uint32> ResultType;
+
+	// Add the required unreal components
+	ResultType.Add(SpatialConstants::UNREAL_METADATA_COMPONENT_ID);
+	ResultType.Add(SpatialConstants::SPAWN_DATA_COMPONENT_ID);
+	ResultType.Add(SpatialConstants::NETMULTICAST_RPCS_COMPONENT_ID_LEGACY);
+	ResultType.Add(SpatialConstants::MULTICAST_RPCS_COMPONENT_ID);
+	ResultType.Add(SpatialConstants::RPCS_ON_ENTITY_CREATION_ID);
+
+	// TODO: breaks without this, but shouldn't need it. Figure out why
+	ResultType.Add(SpatialConstants::ENTITY_ACL_COMPONENT_ID); 
```

Comment:
Entity ACL isn't like position with constant expensive updates, so if the large result type isn't too much of a performance hit by itself, I think this can be tackled as a separate task.